src/features/commands/stats.js

```diff
@@ -0,0 +1,223 @@
+//
+// Written by Su386.
+// See LICENSE for copyright and license notices.
+//
+
+const { EmbedBuilder, SlashCommandBuilder } = require('discord.js');
+const config = require("../../config/config.json")
+const { Octokit } = require('@octokit/rest');
+
+const octokit = new Octokit({
+  auth: process.env.GITHUB_TOKEN
+})
+
+const subcommands = {
+    downloads: { name: "downloads", function: handleDownloadsCommand, permission: false },
+}
+
+let items = []
```

Comment:
```suggestion
let versions = []
```
Don't forget to rename the others to versions

Reply:
Fixed